src/sass/_variables.scss

```diff
@@ -0,0 +1,9 @@
+//fonts
+$header-font: 'Rubik Mono One', sans-serif;
+$body-font: 'Kulim Park', sans-serif;
+
```

Comment:
You could potentially also create some font size variables too if you are reusing some of the same rems throughout!